.github/stale.yml

```diff
@@ -0,0 +1,17 @@
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 180
+# Number of days of inactivity before a stale issue is closed
+daysUntilClose: 7
+# Issues with these labels will never be considered stale
+exemptLabels:
+  - keep
```

Comment:
You should also update this file: https://github.com/sonata-project/dev-kit/blob/master/config/dev-kit.yml

Reply:
It's updated now.